### PR TITLE
update cobrakbase 0.2.0

### PR DESCRIPTION
**Pull request by Fxe**
updated cobrakbase 0.2.0
changed to KBaseFBAModelToCobraBuilder

### CODE REVIEW

File: lib/kb_cobrapy/kb_cobrapyImpl.py.bak-2019-07-25-22-19-11 [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
```
Comment (Tianhao-Gu) [inline lib/kb_cobrapy/kb_cobrapyImpl.py.bak-2019-07-25-22-19-11]: add lib/kb_cobrapy/kb_cobrapyImpl.py.bak* to `.gitignore`

File: lib/kb_cobrapy/Utils.py [hunk 2]
```diff
@@ -36,13 +39,29 @@ def validate_params(params, expected, opt_param=set()):
     def _ws_obj_to_cobra(self, ref):
         ret = self.dfu.get_objects({'object_refs': [ref]})['data'][0]
         name = ret['info'][1]
-        model = cobrakbase.convert_kmodel(ret['data'])
-
+        
+        #old nasty method
+        #model = cobrakbase.convert_kmodel(ret['data'])
```
Comment (Tianhao-Gu) [inline lib/kb_cobrapy/Utils.py]: unused code can be removed.

File: kbase.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.0.3
+    0.0.4
 
 owners:
     [jjeffryes, tgu2]
```
Comment (Tianhao-Gu) [inline kbase.yml]: you can add yourself as owner so that you can register module

File: lib/kb_cobrapy/Utils.py (hunk 2 above)
Comment (Fxe) [reply lib/kb_cobrapy/Utils.py]: I'll leave it for this version, just for easy revert

File: lib/kb_cobrapy/kb_cobrapyImpl.py.bak-2019-07-25-22-19-11 (hunk 1 above)
Comment (Fxe) [reply lib/kb_cobrapy/kb_cobrapyImpl.py.bak-2019-07-25-22-19-11]: updated

File: kbase.yml (hunk 3 above)
Comment (Fxe) [reply kbase.yml]: done

**Pull request merged**